src/utils/Parameters3DUtils.js

```diff
@@ -0,0 +1,32 @@
+/**
+ *  Utilities functions related to 3-D parameter matching
+ *
+ */
+define(function (require, exports, module) {
+    "use strict";
+
+    var PARAMETERS_3D_REGEX = /( )*(((-)?\d+(\.\d+)?)( )+)*((-)?\d+(\.\d+)?( )*)/g;
+    var TRANSLATE_STRING = "position";
+    var ROTATE_STRING = "rotation";
+    var SCALE_STRING = "scale";
+    var RADIUS_STRING = "radius";
+    var MODIFIER = [];
```

Comment:
It would be a bit more common to make this an Object, and use your strings as keys:

```js
var MODIFIERS = {
    position: 50,
    rotation: 30,
    ...
};
```
I think `MODIFIERS` plural also makes more sense.